api/src/test/java/org/openmrs/scheduler/SchedulerServiceTest.java

```diff
@@ -413,4 +414,22 @@ public void shouldSaveLastExecutionTime() throws Exception {
 		assertEquals("Last execution time in seconds is wrong", actualExecutionTime / 1000, td.getLastExecutionTime()
 		        .getTime() / 1000, 1);
 	}
+
+	/**
+	 * @see org.openmrs.scheduler.SchedulerService#getTaskByUuid(java.lang.String)
+	 */
+	@Test
+	public void getTaskByUuid_shouldGetTaskByUuid() throws Exception {
+		TaskDefinition td = Context.getSchedulerService().getTaskByUuid("1365a6da-6493-4e9b-b950-5af1b392aaa3");
+		assertNotNull(td);
+	}
+
+	/**
+	 * @see org.openmrs.scheduler.SchedulerService#getTaskByUuid(java.lang.String)
+	 */
+	@Test
+	public void getTaskByUuid_shouldReturnNullWhenUuidDoesNotExist() throws Exception {
+		TaskDefinition td = Context.getSchedulerService().getTaskByUuid("kncsjvcjvbevismcvbsnksndcsjbvjhvbn");
+		assertNull(td);
```

Comment:
Thanks for the good work done
Why do you use a declaration, if it were me i would use the code below.
`assertNull(Context.getSchedulerService().getTaskByUuid("kncsjvcjvbevismcvbsnksndcsjbvjhvbn");`